x/permission/keeper/keeper.go

```diff
@@ -73,17 +74,40 @@ func (k Keeper) AddGroupMember(ctx sdk.Context, groupID math.Uint, member sdk.Ac
 	id := k.groupMemberSeq.NextVal(store)
 	store.Set(memberKey, id.Bytes())
 	store.Set(types.GetGroupMemberByIDKey(id), k.cdc.MustMarshal(&groupMember))
+
+	// add to expire queue
+	if expiration != nil {
```

Comment:
Can you confirm with Dylan that if `expiration == nil`, it means never expire?

Reply:
Yes. I have confirmed